site/src/content/docs/ref/init-package.mdx

```diff
@@ -104,6 +104,35 @@ The `registry:2` image and the Zarf Agent image can be configured with a custom
 
 :::
 
+## Making the Registry Highly-Available
+
+By default, the registry included in the init package creates a ReadWriteOnce PVC and is only scheduled to run on one node at a time. This setup is usually enough for smaller and simpler deployments. However, for larger deployments or those where nodes are frequently restarted or updated, you may want to make the registry highly-available.
+
+This approach requires certain prerequisites, such as a storage class that supports ReadWriteMany, or being in an environment that allows you to configure the registry to use an S3-compatible backend. Additionally, you must provide custom configuration to the registry to ensure it is distributed across all nodes and has the appropriate number of replicas. Below is an example [configuration file](/ref/config-files/) using a ReadWriteMany storage class:
+
+```yaml
+# zarf-config.yaml
+package:
+    set:
```

Comment:
I think we will need `deploy` in here

https://github.com/defenseunicorns/zarf/blob/b4201718aa96f34f37dac7afa5ccfe59affedbd0/examples/config-file/zarf-config.yaml#L10-L13

Reply:
thats mb, looks like my `suggest` diff didnt sync properly